src/managers/KeybindManager.cpp

```diff
@@ -828,6 +830,19 @@ void CKeybindManager::clearKeybinds() {
 }
 
 void CKeybindManager::toggleActiveFloating(std::string args) {
+    return toggleActiveFloatingCore(args, 0);
+}
+
+void CKeybindManager::setActiveFloating(std::string args) {
+    return toggleActiveFloatingCore(args, 1);
+}
+
+void CKeybindManager::setActiveTiled(std::string args) {
+    return toggleActiveFloatingCore(args, 2);
+}
+
+// forceFloat: 0 for no force, 1 for force float, 2 for force tile
```

Comment:
hm, though this I don't like. Why not just `std::optional<bool>`? more clear tbh.

Reply:
It built but not sure the syntax is correct there, lmk